app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,41 @@
+module StripeWrapper
+  class Charge
+    attr_reader :response, :status
+
+    def initialize(response, status)
+      @response = response
+      @status = status
+    end
+
+    def self.create(options={})
+      StripeWrapper.set_api_key
```

Comment:
I would move this to the initializer because you can most likely have another class than `Charge` like `Customer` as you will see in next lesson, which will need to set the API key. So the best option is to set it right when your Rails app starts and for that case we use initializers.